coinbase.py

```diff
@@ -0,0 +1,20 @@
+import json
+import urllib2
+
+def query(currency,direction):
+  url = "https://api.coinbase.com/v2/prices/" + currency + "-USD/" + direction
+  response = urllib2.urlopen(url, timeout = 5)
+  return json.load(response)
+
+def get_prices(download=True):
+  prices = {}
+  prices['bid'] = {}
+  prices['ask'] = {}
+  curs = ['btc', 'eth', 'ltc', 'bch']
+  for c in curs:
+    tmp = query(c.upper(),"sell")
```

Comment:
Why do we get `sell` prices when `bidding` ? Should it not be `ask` ?

Reply:
sell price is the price one can sell at, so passive bid price
in my dictionary, bid and ask are the passive prices